model/trace/database.yaml

```diff
@@ -141,79 +128,81 @@ groups:
     type: span
     extends: db
     brief: >
-      Call-level attributes for MongoDB
+      Attributes for MongoDB
     attributes:
       - ref: db.mongodb.collection
         requirement_level: required
-        tag: call-level-tech-specific
+        tag: tech-specific
 
   - id: db.elasticsearch
     type: span
     extends: db
     brief: >
-      Call-level attributes for Elasticsearch
+      Attributes for Elasticsearch
     attributes:
       - ref: http.request.method
         requirement_level: required
-        tag: call-level-tech-specific
+        tag: tech-specific
       - ref: db.operation
         requirement_level: required
         brief: The endpoint identifier for the request.
         examples: [ 'search', 'ml.close_job', 'cat.aliases' ]
-        tag: call-level-tech-specific
+        tag: tech-specific
       - ref: url.full
         requirement_level: required
         examples: [ 'https://localhost:9200/index/_search?q=user.id:kimchy' ]
-        tag: call-level-tech-specific
+        tag: tech-specific
       - ref: db.statement
         requirement_level:
           recommended: >
             Should be collected by default for search-type queries and only if there is sanitization that excludes
             sensitive information.
         brief: The request body for a [search-type query](https://www.elastic.co/guide/en/elasticsearch/reference/current/search.html), as a json string.
         examples: [ '"{\"query\":{\"term\":{\"user.id\":\"kimchy\"}}}"' ]
-        tag: call-level-tech-specific
+        tag: tech-specific
       - ref: server.address
-        tag: call-level-tech-specific
+        tag: tech-specific
       - ref: server.port
-        tag: call-level-tech-specific
+        tag: tech-specific
       - ref: db.elasticsearch.cluster.name
         requirement_level:
           recommended: >
             When communicating with an Elastic Cloud deployment, this should be collected from the "X-Found-Handling-Cluster" HTTP response header.
-        tag: call-level-tech-specific
+        tag: tech-specific
       - ref: db.elasticsearch.node.name
         requirement_level:
           recommended: >
             When communicating with an Elastic Cloud deployment, this should be collected from the "X-Found-Handling-Instance" HTTP response header.
-        tag: call-level-tech-specific
+        tag: tech-specific
       - ref: db.elasticsearch.path_parts
         requirement_level:
           conditionally_required: when the url has dynamic values
-        tag: call-level-tech-specific
+        tag: tech-specific
 
   - id: db.sql
     type: span
     extends: 'db'
     brief: >
-      Call-level attributes for SQL databases
+      Attributes for SQL databases
     attributes:
       - ref: db.sql.table
-        tag: call-level-tech-specific
+        tag: tech-specific
+      - ref: db.jdbc.driver_classname
+        tag: tech-specific
```

Comment:
I'd support removing this attribute in a follow-up (we are not capturing it in Java)

Reply:
created https://github.com/open-telemetry/semantic-conventions/issues/782